src/data-storage/insert-trade.ts

```diff
@@ -0,0 +1,29 @@
+import { PrismaClient, Trade } from '@prisma/client';
+import { CreateTradeInput } from '../types';
+
+export const insertTrade = async (trade: CreateTradeInput) => {

```

Comment:
Flavor: Instead of referencing `trade.` with every single line, we could deconstruct the properties here to make it a bit cleaner.

Note: This one is a lot more flavor than the other ones. I personally prefer how deconstruction looks and how assigning key value pairs with just the variable looks.